itr.html

```diff
@@ -0,0 +1,102 @@
+---
+layout: idr
+title: ITR
+---
+
+<script>
+    const LOGOS = {{ site.data.logos | jsonify }};
+    const TAGS = {{ site.data.tags | jsonify }};
+    const TOOLS = {{ site.data.tools | jsonify }};
+    const BIIS_LOGO_URL = "{{"/img/itr/logo_biis.png" | prepend: site.baseurl }}?{{ site.time | date: '%s%N' }}";
+</script>
+
+<header class="marketing-hero">
+    <div class="row homepage text-center">
+        <div class="medium-12 columns">
+            <h1 class="hero-main-header">ITR</h1>
+            <p class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offset-1 large-offset-1">
+                Image Tools Resource (ITR) is an index of suggested open-source
+                analysis software for bioimaging. Though a work-in-progress,
+                the aim is to publish tools that have links to accessible
+                datasets, benchmarking resources, and public workflows to aid
+                in training and reproducibility.
+            </p>
+        </div>
+    </div>
+</header>
+
+<hr class="invisible">
+<div class="row">
+    <div class="small-12 small-centered medium-12 medium-centered columns">
+        <div class="row horizontal">
+            <div>
+                <p>Below you will find community driven
```

Comment:
Maybe have a bit of a left/right margin for this text, for me it goes right to the edge.